CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ Which is based on [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
 
+## [2.5.0] - 2018-04-06
```

Comment:
We version the gem and changelog post acceptance for several reasons:
- PRs are not a FIFO queue nor do they have an SLA on review times so doing this when we are ready to release prevents us needing to ask for additional rebases
- maintainer may disagree with you on the impact (versioning is hard)
- we date it from when it was released rather than when it was submitted

https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md#unreleased-change